compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs

```diff
@@ -11,9 +11,9 @@ use rustc_span::source_map::DesugaringKind;
 use rustc_span::symbol::{kw, Symbol};
 use rustc_span::Span;
 
-use crate::borrow_check::diagnostics::BorrowedContentSource;
-use crate::borrow_check::MirBorrowckCtxt;
-use crate::util::collect_writes::FindAssignments;
+use crate::diagnostics::BorrowedContentSource;
+use crate::MirBorrowckCtxt;
+use rustc_const_eval::util::collect_writes::FindAssignments;
```

Comment:
The single impl of this trait is very small, yet not marked as `#[inline]`. Same applies for the `FindLocalAssignmentVisitor` it uses.